migrations/Version20180626065141.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20180626065141 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
```

Comment:
What about deleting this comment and the one placed in down() method as well? :D

Reply:
It's placed in every SyliusStandard migration, so even though it's quite silly, I would live it for now :)